.env.example

```diff
@@ -1,6 +1,8 @@
 # Shared Environment
 LOCAL_SEED_ADDRESS=0xYOUR_DEV_ADDRESS
 HARDHAT_OWNER_ADDRESS=0xYOUR_DEV_ADDRESS
+HARDHAT_FORK_BLOCK=13500000
+HARDHAT_GANACHE_PORT=8555
 REACT_APP_INFURA_PROJECT_ID=infuraidhere
```

Comment:
I'm getting this error: 
"Starting RPC server
CodedError: project ID does not have access to archive state"

which makes sense because I passed in a free Infura ID. I assume the forking requires "archive" data access which will require a paid INFURA id. 

Anyhow, maybe we document this more clearer, and maybe there's a way to run some of the tests without needing the fork?

Reply:
https://infura.io/docs/ethereum/add-ons/archiveData

Reply:
As-is, the tests are pretty dependent on the fork. I'll explore reducing this dependency in the future.

Reply:
Added a line in the README about needing access to archive data